### PR TITLE
first attempt at a socials post

### PR DESCRIPTION
**Pull request by scottcain**

### CODE REVIEW

**Review by wm75 — APPROVED** [review]

**Comment by natalie-wa** [comment]
For future social media posts, please try to stick to the character count limit on BlueSky. The (1/2) and (2/2) posts tend to look messy, and we should prioritize linking to more information instead of trying to crunch it all into one post or a string of posts that are difficult to read and digest.

**Comment by wm75** [comment]
Sorry @natalie-wa, should probably have given you the chance to review as well, just didn't want to keep this lying around until tomorrow.
Should we make it the practice to have you review stuff that's going to any of the Galaxy project's accounts?

**Comment by scottcain** [comment]
Yeah, 300 characters is tough! I will do that going forward.

**Comment by scottcain** [comment]
Re: Galaxy Hub. Yes! Is this something I can learn to do quickly? (Conversation can move to Matrix at this point :-)

**Comment by bgruening** [comment]
@scottcain Long version is here: https://github.com/galaxyproject/galaxy-hub/blob/master/CONTRIBUTING.md

tldr; drop a folder any file in here: https://github.com/galaxyproject/galaxy-hub/tree/master/content/events
